src/pdfpc.vala

```
@@ -260,6 +262,109 @@ namespace pdfpc {
             this.controller = new PresentationController( metadata, Options.black_on_end );
             this.cache_status = new CacheStatus();
 
+            // Prerender mode
+            if(Options.prerender_all) {
+                if(!Options.persist_cache) {
+                    print("Note: Using --prerender-all without -p is useless.\n");
```

Print it to stderr and let the program fail